### PR TITLE
Updates to fix things after the conversion to use the PIP installer

### PR DESCRIPTION
**Pull request by madscientist**
See commit messages for details.

### CODE REVIEW

**Comment by madscientist** [comment]
I got a "rate limited" error on the travis builds so... don't know what that's about :-/

**Comment by vegichan** [comment]
We probably tried to download the migrator too many times in a short period of time. https://docs.github.com/en/rest/reference/rate-limit

File: client/pkg/pynuoadmin.py [hunk 1]
```diff
@@ -27,26 +30,38 @@ def __init__(self):
...
     def install(self):
-        self.stage.stage('python', ['./'])
+        # Omit the files that were installed due to pynuodb
+        self.stage.omitcontents = Package.get_package('pynuodb').staged[0].getcontents()
+
+        self.stage.stage(os.path.join('python', 'site-packages'), ['./'])
 
         nuodb = self.get_package('nuodb')
+
+        # This needs to go two levels below the root, as it contains a ../..
```
Comment (vegichan) [inline client/pkg/pynuoadmin.py:52]: a much simpler solution is to fix `pynuoadmin` to follow the usual pip location and install itself in a directory

Comment (madscientist) [reply client/pkg/pynuoadmin.py:52]: Hm.  I considered that, but is it really "simpler"?  We'd have to first make a new pynuoadmin package and post it to pypi, then update this to manage it, and all the current users of pynuoadmin would have to be changed to import the new path.

To me it seems like creating a subdirectory here is actually simpler :)

Comment (vegichan) [reply client/pkg/pynuoadmin.py:52]: we can do this in a separate PR, if we decide to change the structure and bump the major version

Comment (madscientist) [reply client/pkg/pynuoadmin.py:52]: I don't think moving the structure changes anything... it moves things down one level in the DB package too, so we'll still have to keep this extra directory level in the client package...?

**Pull request closed without merging**